jsettlers.logic/src/jsettlers/main/JSettlersGame.java

```diff
@@ -93,6 +97,7 @@ private JSettlersGame(IGameCreator mapCreator, long randomSeed, INetworkConnecto
 		this.networkConnector = networkConnector;
 		this.playerId = playerId;
 		this.availablePlayers = availablePlayers;
```

Comment:
Idea: Change this to a PlayerSetting[] availablePlayers. Then give them an isAvailable() and an isAi() method. In the future, we could add difficulty/... flags here.